### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Justman100**

### CODE REVIEW

**Comment by Justman100** [comment]
@cyberpower678

**Comment by cyberpower678** [comment]
Looks like there's some needed conflict resolution here.

**Review by cyberpower678 — CHANGES_REQUESTED** [review]
Please use an mdash instead

**Review by cyberpower678 — APPROVED** [review]

File: app/src/html/i18n/en.json [hunk 1]
```diff
@@ -1,824 +1,824 @@
...
+  "page_scan": "Pages to scan",
+  "paper": "Paper",
+  "paywall": "Subscription site",
+  "paywall_tagsplaceholder": "This is a comma separated list of templates that indicated the link it's analyzing is behind a paywall.",
+  "paywall_tags": "Paywall tags",
+  "performancemetricsheader": "Performance metrics",
+  "permingroups": "This permission is obtainable with the following groups: {{grouplist}}",
+  "permissionchangeerror": "Permission change failed",
+  "permissionchangeoverstep": "There is a permission or group you are trying to illegally modify in this request.",
+  "permissionchangepermissionchangeglobal": "changed global user permissions for <a href=\"index.php?page=user&id={{targetuserid}}\">{{targetusername}}</a>! <b>Added:</b> {{logfrom}}; <b>Removed:</b> {{logto}}",
```
Comment (cyberpower678) [inline app/src/html/i18n/en.json:551]: Please use an mdash for separation.  ! are not appropriate for log entries.

File: app/src/html/i18n/en.json [hunk 2]
```diff
@@ -1,824 +1,824 @@
...
+  "permissionchangepermissionchange": "changed user permissions for <a href=\"index.php?page=user&id={{targetuserid}}\">{{targetusername}}</a>! <b>Added:</b> {{logfrom}}; <b>Removed:</b> {{logto}}",
```
Comment (cyberpower678) [inline app/src/html/i18n/en.json:552]: Please use an mdash for separation.  ! are not appropriate for log entries.

**Review by cyberpower678 — CHANGES_REQUESTED** [review]

**Comment by cyberpower678** [comment]
Please correct my review and perform a conflict resolution against the current en.json file.  10 other pull requests were merged in before yours.

**Comment by amire80** [comment]
Messages in en.json must not be sorted alphabetically by key. They should be sorted by groups of where they appear in the interface to make translation convenienet.

**Comment by Justman100** [comment]
> Please correct my review and perform a conflict resolution against the current en.json file. 10 other pull requests were merged in before yours.

A ! will get there! It is the end of the sentence!

**Pull request closed without merging**